docs/debian12-install-guide.md

```diff
@@ -80,6 +110,7 @@ sudo mysql_secure_installation
 - Disallow root login remotely:  Y
 - Remove test database:          Y
 - Reload privilege tables:       Y
+
 ### Setup SQL Database
 ```bash
 sudo mysql -u root -p
```

Comment:
Change to just `sudo mysql` and remove instruction to enter password. (same [here](https://github.com/azerothcore/wiki/pull/1023/files#diff-d1f2d59a1e6c86993f0dadd798a430a0cb71f8cf13ff37eee9871b73dbb3478fR227-R230))